source/refunding_payments/index.html.md.erb

```diff
@@ -7,9 +7,9 @@ weight: 4100
 
 # Refund a payment
 
-You can refund all or part of a payment using the GOV.UK Pay API.
+<%= warning_text('If you are a member of the public trying to get a refund, please contact the government department you made the payment to. This page is for civil servants that take payments with GOV.UK Pay. ') %>
```

Comment:
I wonder about "trying to get a refund" - it sounds a bit like they're fighting/failing against the government machine? Maybe "If you are a member of the public requesting...?"

Reply:
Very good point, comrade. I'll use your suggestion.